lib/typescript.js

```diff
@@ -60,5 +58,20 @@ module.exports = {
     "@typescript-eslint/no-confusing-non-null-assertion": "off",
     "@typescript-eslint/prefer-for-of": "off",
     "@typescript-eslint/prefer-function-type": "off",
+
+    // rules deleted from `recommended` in v8
+    "no-loss-of-precision": "off",
+    "@typescript-eslint/no-loss-of-precision": "error",
+
+    // added instead of `@typescript-eslint/ban-types` in v8
+    "@typescript-eslint/no-empty-object-type": "error",
+    "@typescript-eslint/no-unsafe-function-type": "error",
+    "@typescript-eslint/no-wrapper-object-types": "error",
+
+    // rules added to `recommended` in v8
+    "@typescript-eslint/no-require-imports": "off",
+    "no-unused-expressions": "error",
+    "@typescript-eslint/no-unused-expressions": "off",
+    "@typescript-eslint/prefer-namespace-keyword": "off",
```

Comment:
![ask](https://img.shields.io/badge/review-ask-blue.svg)
How to check the difference in the rule set?

Reply:
I basically used the following page as a reference
- https://typescript-eslint.io/blog/announcing-typescript-eslint-v8/#updated-configuration-rules

Also, as for `ban-types`, the rules have been split into multiple rules, so I have enabled each of the split rules.
- https://typescript-eslint.io/rules/ban-types/